blocks/aside/aside.js

```diff
@@ -165,11 +159,14 @@ export default async function decorate(block) {
     );
 
     pdfButton.addEventListener('click', async () => {
-      // Add the js2pdf script
-      await loadScript('https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.5.1/jspdf.umd.min.js');
-      await loadScript('/scripts/html2canvas.min.js');
-      if (window.jspdf) {
-        await generatePDF(pageName);
+      // Add the html2pdf script
+      await loadScript('https://cdnjs.cloudflare.com/ajax/libs/html2pdf.js/0.10.1/html2pdf.bundle.min.js', {
+        integrity: 'sha512-GsLlZN/3F2ErC5ifS5QtgpiJtWd43JWSuIgh7mbzZ8zBps+dvLusV+eNQATqgA/HdeKFVgA5v3S/cIrLF7QnIg==',
+        crossorigin: 'anonymous',
+        referrerpolicy: 'no-referrer',
+      });
+      if (window.html2pdf) {
+        await generatePDF(pageTitle);
```

Comment:
@amol-anand `pageTitle` may have special chars that are not allowed in file system, can we use pageName here?

Reply:
good point. yes will update to pageName instead.